boms/preview-ee/pom.xml

```diff
@@ -51,7 +51,7 @@
         <version.jakarta.websocket.jakarta-websocket-api>2.2.0</version.jakarta.websocket.jakarta-websocket-api>
         <version.jakarta.ws.rs.jakarta-ws-rs-api>4.0.0</version.jakarta.ws.rs.jakarta-ws-rs-api>
         <version.org.glassfish.expressly>6.0.0-M1</version.org.glassfish.expressly>
-        <version.org.glassfish.jakarta.faces>4.1.0</version.org.glassfish.jakarta.faces>
+        <version.org.glassfish.jakarta.faces>4.0.9</version.org.glassfish.jakarta.faces>
```

Comment:
Version 4.1.0 is a Jakarta EE 11 version. If Preview is target EE 11, then we likely don't want this change.

Reply:
Thanks @jasondlee. I'll close this then and we'll need to do 4.0.9 in std WF ourselves (or not).